chart/compass/charts/director/templates/configmap-tenant-config.yaml

```diff
@@ -0,0 +1,13 @@
+{{- if eq .Values.global.tenantConfig.externalTenants false }}
```

Comment:
it looks strange that if externalTenants is set to false, we create some configmap, maybe value should be called static configuration?